plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/CheckmarxESReader.java

```diff
@@ -116,22 +111,23 @@ private boolean isIrrelevant(String name) {
                 || name.equals("Unprotected_Cookie");
     }
 
-    private int translate(int cwe) {
+    private CweNumber translate(int cwe) {
         switch (cwe) {
             case 77:
             case 15:
-                return CweNumber.COMMAND_INJECTION;
+                return CweNumber.OS_COMMAND_INJECTION;
             case 36:
             case 23:
                 return CweNumber.PATH_TRAVERSAL;
             case 338:
                 return CweNumber.WEAK_RANDOM;
         }
-        return cwe;
+
```

Comment:
I think we should add a warning like the others, e.g., "something new seen here not mapped before". Take exact message from other parser that already has this warning.

Reply:
As discussed: https://github.com/OWASP-Benchmark/BenchmarkUtils/pull/31#issuecomment-1304863379